scripts/measures/2023/update-measures-util.ts

```diff
@@ -0,0 +1,239 @@
+/**
```

Comment:
New file added to break logic out of the primary script. This helps with testing.